moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -107,9 +165,26 @@ const NewProjectMap = ({
     hoveredCoords,
   } = useHoverLayer();
 
-  const { visibleLayerIds, renderLayerSelect, mapStyleConfig } = useLayerSelect(
-    getLayerNames(),
-    classes
+  const {
+    visibleLayerIds,
+    renderLayerSelect,
+    mapStyleConfig,
+    handleBasemapChange,
+    mapStyle,
+  } = useLayerSelect(getLayerNames(), classes);
+
+  /**
+   * Creates a geojson layer with all the other features of the project
+   * and converts the feature type into whatever we specify. In this case,
+   * we will use 'projectFeatures'.
+   * @type {Object} otherProjectFeaturesCollection - GeoJSON Collection
+   */
+  const otherProjectFeaturesCollection = useTransformProjectFeatures(
+    projectFeatureCollection,
+    {
+      LineString: "projectFeatures",
+      Point: "projectFeaturePoints",
+    }
```

Comment:
In here, we use the useTransformProjectFeatures custom hook, along with a previously generated collection to rename all the layers of lines to projectFeatures, and points to projectFeaturePoints.